### DISCUSSION TITLE
【2023.2.1】community meeting notes

### DISCUSSION THREAD

**Discussion by Stevent-fei** [sealerio/sealer#1997] [General]
hi all, 今晚8点，有咱们sealer的开发者会议，大家有什么议题可以提前准备哦。

**Comment by jsparter** [comment]
#1940 use zot to replace inbuild registry

**Comment by zhy76** [comment]
#1966 Add more E2E test cases for sealer

**Comment by starnop** [comment]
https://github.com/sealerio/sealer/issues/2001 0.9.2 release KO

**Comment by YTGhost** [comment]
Use Sealer Image Offline Document

**Comment by kakaZhou719** [comment]
2月1日 【会议纪要】


Starnop:
1. 同步0.9.2 release ko

jsparter: sealer 使用 oci registry 的可行性讨论。
1. oci registry 对sealer run 的性能影响？zot vs docker registry 最好也能体现到proposal中。
2. oci registry 是否支持 helm package？
3. 对skopeo 是否有sdk 使用？
4. 是否可以在容器中启动zot？
5. 是否影响到ha mode？
6. 下载容器镜像的代码层面如何优雅的集成。


zhy76:同步sealer e2e 开发进展
1. 需要优先保证e2e测试框架稳定，包括infra镜像，GitHub action 配置。
2. 需要对测试case 进行分类，优先级保证主流程覆盖。
3. 需要详细的开发和使用文档，方便更多同学参与。
3. 优化点: 并行执行E2E, 或者更够粒度更细的执行case。


YTGhost: 同步kyverno开发进展，以及使用文档。
1. 需要0.9.1的非hack docker 的k8s底座。
2. 读取yaml中的image: 检测image 地址合法性。
3. 增加ignore image list flag 在sealer build 中支持用户反向选择需要下载的容器镜像。
4. sealer build ,如果判断不出app type,用户需要在kubefile中 写上appcmds。
5. 启动kyverno时候获取默认的registry domain，sealer 安装的底座来自于clusterfile。
6. kyverno文档中增加run章节，保证用户的使用体验。
